phpBB/phpbb/event/dispatcher.php

```diff
@@ -40,4 +46,33 @@ public function trigger_event($eventName, $data = array())
 		$this->dispatch($eventName, $event);
 		return $event->get_data_filtered(array_keys($data));
```

Comment:
trigger_event() use dispatch() to really dispatch the event, so it should be good